modules/apps/frontend-editor/frontend-editor-ckeditor-web/src/main/resources/META-INF/resources/ckeditor.jsp

```diff
@@ -625,7 +625,7 @@ name = HtmlUtil.escapeJS(name);
 		});
 
 		ckEditor.on('dataReady', (event) => {
-			if (instancePendingData) {
+			if (instancePendingData != null) {
```

Comment:
Not sure about the overall solution, but in any case we should **always** use strict equality, i.e. `!==` and not `!=`

Reply:
Okay, that makes sense, I will do the change.

Regards